packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationProfileNavbar.tsx

```diff
@@ -33,7 +33,7 @@ export const OrganizationProfileNavbar = (
               name: 'Drivepoint',
               logoUrl: 'https://images.lclclerk.com/uploaded/img_2GE5fixIVWTNMdRjcH7ivtelSxT.png',
             }}
-            sx={t => ({ margin: `0 0 ${t.space.$2x5} ${t.space.$2}` })}
+            sx={t => ({ margin: `0 0 ${t.space.$2x5} ${t.space.$2}`, maxWidth: t.space.$60 })}
```

Comment:
Looks good!
How did you end up with this fixed width? Is this the same as the components in figma?

Reply:
It's the same as the starting width of the `Navbar`. I wanted it (the `Navbar`) to remain flexible in case we need some longer text in the routes. Do you think we should add a `max-width` to the `OrganizationPreview` itself or the `Navbar` instead?